README.md

```diff
@@ -1,12 +1,12 @@
-# `@ubiquity-os/telegram-bot`
+# `@ubiquity-os/telegram-bridge`
```

Comment:
I just realized but isn't this considered a new kernel? Does it make sense to call it `ubiquity-os-kernel-telegram`?